src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpSyntaxFormattingOptions.cs

```diff
@@ -61,6 +61,7 @@ internal sealed record class CSharpSyntaxFormattingOptions : SyntaxFormattingOpt
     [DataMember] public bool WrappingPreserveSingleLine { get; init; } = true;
     [DataMember] public CodeStyleOption2<NamespaceDeclarationPreference> NamespaceDeclarations { get; init; } = s_defaultNamespaceDeclarations;
     [DataMember] public CodeStyleOption2<bool> PreferTopLevelStatements { get; init; } = s_trueWithSilentEnforcement;
+    [DataMember] public int CollectionExpressionWrappingLength { get; init; } = 120;
```

Comment:
consider defining the constant in one place so it isn't duplicated in two locations.

Reply:
Yup, missed that.